io.neovim.nvim.appdata.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Neovim contributors -->
+<component type="desktop-application">
+  <id>io.neovim.nvim</id>
+  <translation type="gettext">nvim</translation>
+  <name>Neovim</name>
+  <summary>Vim-fork focused on extensibility and usability</summary>
+  <description>
+    <p>
+      Neovim is a refactor of the Vim text editor engineered to encourage new features, advanced UIs, and powerful extensions.
+    </p>
+  </description>
+  <kudos>
+    <kudo>UserDocs</kudo>
+  </kudos>
+  <project_license>Apache-2.0</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://neovim.io/</url>
+  <url type="bugtracker">https://github.com/neovim/neovim/issues</url>
+  <url type="faq">https://github.com/neovim/neovim/wiki/FAQ</url>
+  <url type="help">https://neovim.io/doc/</url>
+  <url type="donation">https://neovim.io/sponsors/</url>
+  <url type="translate">https://github.com/neovim/neovim/tree/master/src/nvim/po</url>
+  <content_rating type="oars-1.1"/>
+  <releases>
+    <release date="2018-07-19" version="0.3.1"/>
+    <release date="2018-07-11" version="0.3.0"/>
+  </releases>
+  <launchable type="desktop-id">nvim.desktop</launchable>
```

Comment:
Does this works in the same way `io.neovim.nvim.desktop` would be?

Reply:
It gets renamed during along with the desktop file.

Reply:
Of course it should be renamed upstream too.